rust/src/lib/nm/query_apply/profile.rs

```diff
@@ -192,7 +192,7 @@ fn _activate_nm_profiles(
     }) {
         if let Some(uuid) = nm_conn.uuid() {
             log::info!(
-                "Activating connection {}: {}/{}",
+                "Reapplying connection {}: {}/{}",
```

Comment:
What is the reason changing to `Reapplying connection` here ?

Reply:
To distinguish activation vs  reapply.